Microsoft.Toolkit.Uwp.UI.Controls.Graph/PowerBIEmbedded/EmbedReportFrame.html

```diff
@@ -0,0 +1,213 @@
+﻿<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title></title>
+    <style>
+        html, body {
+            border: 0px;
+            margin: 0px;
+            height: 100%;
+            font-family: 'Segoe UI Web Light', 'Segoe UI Light', 'Segoe WP Light', 'Segoe UI', 'Segoe WP', Tahoma, Arial, sans-serif;
+            overflow: hidden;
+        }
+        #dropdownContainer {
+            position: absolute;
+            left: 10px;
+            top: 10px;
+        }
+        #dropdownContainer select {
+            margin-right: 15px;
+            padding: 3px;
+            font-size: 1em;
+        }
+        #reportContainer {
+            height: 100%;
+        }
+
+        @media (max-width: 720px) {
+            #dropdownContainer {
+                position: static;
+            }
+            #dropdownContainer select {
+                width: 100%;
+            }
+        }
+    </style>
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+    <script src="https://npmcdn.com/es6-promise@3.2.1"></script>
```

Comment:
Chakra supports ES6, so this shouldn't be needed.

Reply:
removed